ucm2/Intel/sof-essx8336/Hdmi.conf

```diff
@@ -0,0 +1,62 @@
+SectionDevice."HDMI1" {
+	Comment "HDMI/Display Port 1 Stereo"
+
+	Value {
+		PlaybackPCM "hw:${CardId},5"
+		PlaybackPriority 500
+		If.1 {
```

Comment:
EnableSequence [
				cset "name='IEC958 Playback Switch' on"
			]

			DisableSequence [
				cset "name='IEC958 Playback Switch' off"
			]

Enable sequence needed because of a bug in Ubuntu

Reply:
I am not sure why @yangxiaohua2009 , we never set this IEC control and things worked fine so far. What bug are you referring to?

Reply:
You may refer to [hda/hdmi.conf](https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/codecs/hda/hdmi.conf).
Ubuntu turn off the IEC958 switch automatically which causes no sound in HDMI.